app/assets/markdown/cs1.md

```diff
@@ -341,7 +341,9 @@ Sample Response:
 #### _(Levels 15 - 17)_
 ### Summary
 
-Working with variables is like organizing things in shoeboxes. You give the shoebox a name, like "School Supplies", and then you put things inside. The exact contents of the box might change over time, but whatever's inside will always be called "School Supplies". In programming, variables are symbols used to store data that will change over the course of the program. Variables can hold a variety of data types, including numbers and strings. In many programming languages, we need to declare the type of data a variable can store before using it. For example, in JavaScript, a variable named “age” which stores the value 16, has to be declared with the integer data type, by preceding the assignment statement with the word “int”, to not result in errors. It would look like this:
+Working with variables is like organizing things in shoeboxes. You give the shoebox a name, like "School Supplies", and then you put things inside. The exact contents of the box might change over time, but whatever's inside will always be called "School Supplies". In programming, variables are symbols used to store data that will change over the course of the program. Variables can hold a variety of data types, including numbers and strings. In many programming languages, we need to declare the type of data a variable can store before using it. For example, in Java, a variable named “age” which stores the value 16, has to be declared with the integer data type, by preceding the assignment statement with the word “int”, to not result in errors. It would look like this: 
+
+int age = 16;
```

Comment:
To get code formatting, it would be better to wrap it in triple backticks, with the first one starting with the programming language. Imagine this but with backticks instead of apostrophes:

```
'''java
int age = 16;
'''
```